Documentation/community/roadmap.rst

```diff
@@ -72,43 +97,28 @@ Cilium has applied for `CNCF Graduation`_, please add your support on the PR!
 Cilium Service Mesh 
 ~~~~~~~~~~~~~~~~~~~
 
-Our eBPF-accelerated Service Mesh is the main focus for
-major enhancement, and it's a natural evolution of Cilium's networking
-capabilities. We released a beta at the end of 2021 and had very valuable
-feedback from our user community. The next steps we'd like to take for Cilium
-Service Mesh (in no particular order) are: 
+As Cilium Service Mesh gets wider adoption, the next steps we'd like to take in
+this area (in no particular order) are: 
 
 * Graduating Prometheus metrics and OpenTelemetry collector to stable
-* Using Kubernetes as service mesh control plane 
- 
-  * Simple to use sidecar-free service mesh configured using Kubernetes Services
-    and Ingress with support for additional annotations
-
-* Graduating EnvoyConfig CRD to stable
```

Comment:
Was this dropped because we abandoned making EnvoyConfig stable? Because I'm not sure we went through any process to decide that it's stable.

Reply:
Yes, I think that we're better to keep EnvoyConfig as _mostly_ an implementation detail that brave folks may make use of if they're game. And then find things that people want to do with their own EnvoyConfigs and do them in Gateway API instead.

Reply:
I took these out because I think they're largely covered by the table above (if I correct GW API and L7 traffic management to be marked as Beta). But you're right, I should leave something here about graduating both to stable

Reply:
Added bullets for graduating Gateway API, L7 Traffic Management, next-gen mutual authn and SPIFFE integration